ReadmeSrc/Readme.js

```diff
@@ -0,0 +1,116 @@
+require('dotenv').load()
+const axios = require('axios')
+const moment = require('moment')
+const merge = require('lodash/fp/merge')
+const table = require('markdown-table')
+const delay = require('delay')
+
+const {getRows, headers} = require('./Rows')
+
+const data = require('../webpage/src/data.json')
+
+let rows
```

Comment:
I understand this is one-time-run script and this is not critical, but i prefer to avoid global state. It is hard to read this code. You can encapsulate `let rows` inside function which is using it and latter returns it.

Reply:
That's not global state. 
https://github.com/albinotonnina/mmarkdown/blob/master/src/extract.js#L51

Reply:
This is global state for functions inside of this file. E.g. order function calls matters, calling same function twice can change result of function

Reply:
Anyway yes. Of course I had the same thought of your with that. 
And I agree with you, it's not critical.
But I changed that.